SFR/common_input.i

```diff
@@ -0,0 +1,13 @@
+pitch = 1.25984  # fuel pitch
+height = 100.0  # fuel rod height
+r_fuel = 0.4715  # radius of the fuel rod
+t_gap = 0.0150  # gas gap thickness
+r_clad_inner = 0.4865  # radius of cladding inner
+t_clad = 0.05  # cladding thickness
+edge_length = 12.1705  # side length of the fuel assembly
+lattice_pitch = 1.24  # Pitch used in lattice or mesh generation
```

Comment:
I think Paul may have mentioned this, but you should be careful with units. I think most MOOSE stuff defaults to SI, but these look like `cm` to me. If you aren't careful, your MOOSE and OpenMC geometries may be off by a factor of 100 and the coupling will not work